### ISSUE TITLE
Warning ItemJoin v5.0.7-RELEASE-b537

### ISSUE THREAD

**Issue by naumnek** [RockinChaos/ItemJoin#329]
## Thanks for taking the time to open a bug report!

**Describe the bug**
The plugin ItemJoin v5.0.7-RELEASE-b537crashes when changing items. yml

**To Reproduce**
1. Replace the text in items. yml with
```
   stormbow:
     id: BOW
     slot: Arbitrary
     name: '&6Гнев Грозы'
     lore:
     - '&7Редкость:&e Артефакт'
     - '&7Комплект:&f Наследие Всадника'
     commands:
       interact-left:
       - 'op: execute at @s[level=100..,nbt={Inventory:[{id:"minecraft:dragon_egg"}]}] at @e[distance=..100,type=arrow,limit=10] at @e[type=!arrow,type=!item,name=!%player%,distance=..6] run summon lightning_bolt ~ ~1 ~'
       - 'op: effect give @s[level=100..,nbt={Inventory:[{id:"minecraft:dragon_egg"}]}] regeneration 10 1 true'      
       - 'op: execute at @s[level=100..,nbt={Inventory:[{id:"minecraft:dragon_egg"}]}] at @e[distance=..100,type=arrow,limit=10] run effect give @e[type=!arrow,type=!item,name=!%player%,distance=..6] minecraft:slowness 5 2'      
     commands-sequence: SEQUENTIAL
     commands-item: LAPIS_LAZULI
     commands-cost: 1
     commands-cooldown: 1
     cooldown-message: '&7До окнчания перезарядки %item%&7 осталось &f%timeleft% &7секунд'
     itemflags: item-store, death-drops, self-drops, hide-attributes, unbreakable
     triggers: disabled 	
   scroll1-1:
     id: PAPER
     slot: Arbitrary
     name: '&bСвиток'
     lore:
     - '&7Редкость: Обычная'
     - '&7Свойство:&f Принятие'
     commands:
       interact-right:
       - 'op: minecraft:tag @s[level=30..] add scroll1'  
       - 'delay: 1200'    
       - 'op: minecraft:tag @s remove scroll1'              
     enchantment: DURABILITY:1
     commands-sequence: SEQUENTIAL  
     cooldown-message: '&7До окнчания перезарядки %item%&7 осталось &f%timeleft% &7секунд'
     itemflags: hide-attributes, disposable, item-modifiable, item-craftable, item-repairable
     triggers: disabled 	
     enabled-worlds: ALL	
   scroll1-2:
     id: PAPER
     slot: Arbitrary
     name: '&bСвиток'
     lore:
     - '&7Редкость: Обычная'
     - '&7Свойство:&f Запрос'
     commands:
       interact-right:          
       - 'op: minecraft:tp @p[tag=scroll1]'
       - 'delay: 40'
       - 'op: minecraft:tag @a[distance=..3] remove scroll1'          
     enchantment: DURABILITY:1
     commands-sequence: SEQUENTIAL  
     cooldown-message: '&7До окнчания перезарядки %item%&7 осталось &f%timeleft% &7секунд'
     itemflags: hide-attributes, disposable, item-modifiable, item-craftable, item-repairable
     triggers: disabled 	
     enabled-worlds: ALL
```
**Expected behavior**
How the plugin works

**Screenshots**
[12:04:50 WARN]: [ItemJoin] Plugin ItemJoin v5.0.7-RELEASE-b537 generated an exception while executing task 523451
java.lang.NullPointerException: null
 at me.RockinChaos.itemjoin.utils.Chances$Chance.access$100(Chances.java:174) ~[?:?]
 at me.RockinChaos.itemjoin.utils.Chances.getRandomElement(Chances.java:90) ~[?:?]
 at me.RockinChaos.itemjoin.utils.Chances.getRandom(Chances.java:153) ~[?:?]
 at me.RockinChaos.itemjoin.item.ItemUtilities.handleItems(ItemUtilities.java:216) ~[?:?]
 at me.RockinChaos.itemjoin.item.ItemUtilities.lambda$null$4(ItemUtilities.java:201) ~[?:?]
 at me.RockinChaos.itemjoin.handlers.ServerHandler.lambda$runAsyncThread$0(ServerHandler.java:43) ~[?:?]
 at org.bukkit.craftbukkit.v1_16_R2.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.16.2.jar:git-Paper-161]
 at org.bukkit.craftbukkit.v1_16_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:54) ~[patched_1.16.2.jar:git-Paper-161]
 at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.16.2.jar:git-Paper-161]
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_231]
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_231]
 at java.lang.Thread.run(Thread.java:748) [?:1.8.0_231]
 [12:04:50 WARN]: [ItemJoin] Plugin ItemJoin v5.0.7-RELEASE-b537 generated an exception while executing task 523450
java.lang.NullPointerException: null
 at me.RockinChaos.itemjoin.utils.Chances$Chance.access$100(Chances.java:174) ~[?:?]
 at me.RockinChaos.itemjoin.utils.Chances.getRandomElement(Chances.java:90) ~[?:?]
 at me.RockinChaos.itemjoin.utils.Chances.getRandom(Chances.java:153) ~[?:?]
 at me.RockinChaos.itemjoin.item.ItemUtilities.handleItems(ItemUtilities.java:216) ~[?:?]
 at me.RockinChaos.itemjoin.item.ItemUtilities.lambda$null$4(ItemUtilities.java:201) ~[?:?]
 at me.RockinChaos.itemjoin.handlers.ServerHandler.lambda$runAsyncThread$0(ServerHandler.java:43) ~[?:?]
 at org.bukkit.craftbukkit.v1_16_R2.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.16.2.jar:git-Paper-161]
 at org.bukkit.craftbukkit.v1_16_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:54) ~[patched_1.16.2.jar:git-Paper-161]
 at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.16.2.jar:git-Paper-161]
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_231]
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_231]
 at java.lang.Thread.run(Thread.java:748) [?:1.8.0_231]

**Pastebin**
Even if it is not related to the problem, please include ALL configuration files in separate Pastebin.com links as well as any related error codes from the console window.

**Please complete the following information:**
 - Resource Version and Build Number: [git-Paper-161]
 - Server Version [Paper MC: 1.16.2]

**Additional context**
config.yml:

> config-Version: 7
> 
> Language: 'English'
> 
> General:
>   CheckforUpdates: true
>   Metrics-Logging: true
>   Log-Commands: true
>   Debugging: false
> 
> Database:
>   MySQL: false
>   host: 'localhost'
>   port: 3306
>   table: 'database_name'
>   user: 'root'
>   pass: 'password'
> 
> Settings:
>   HeldItem-Slot: DISABLED
>   HeldItem-Triggers: JOIN, WORLD-SWITCH
>   DataTags: true
> 
> Permissions:
>   Obtain-Items: false
>   Obtain-Items-OP: false
>   Commands-Get: false
>   Commands-OP: false
>   Movement-Bypass: true
> 
> Clear-Items:
>   Type: ITEMJOIN
>   Delay-Tick: 2
>   Join: DISABLED
>   World-Switch: DISABLED
>   Region-Enter: DISABLED
>   Options: PROTECT, PROTECT_OP, PROTECT_CREATIVE
>   Blacklist: '{id:BEDROCK}, {slot:4}, {name:Blazefury}'
> 
> Active-Commands:
>   commands:
>     - 'itemjoin get <item> <player>'
>     - 'first-join: say This is a command only executed once per world, per player.'
>   triggers: disabled 
>   enabled-worlds: DISABLED
> 
> Prevent:
>   Pickups: DISABLED
>   itemMovement: DISABLED
>   Self-Drops: DISABLED
>   Death-Drops: DISABLED
>   Bypass: CREATIVE, OP

**Comment by RockinChaos** [comment]
Can you please provide your full (broken) items.yml in a pastebin.com link? Some additional context is missing that seems to be causing the error.

**Comment by RockinChaos** [comment]
This bug should be fixed in the release of ItemJoin v5.1.0.
You can grab it here; Download: https://www.spigotmc.org/resources/itemjoin.12661/download?version=356084

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes


Please re-open this issue if the problem continues.
